Fix health bar to work with Forge left_height

mccreery opened the pull request:
Single commit PR, just changes the initial value of (local) `left_height` to the global `GuiIngameForge.left_height` so the bar renders at the correct height for compat with other mods. See #114  for an example.

bonii-xx commented:
Oh nice. Please change the PR onto the 1.12 branch. Did you check if there are any other hardcoded values? How fast do you need the fix released?

mccreery commented:
Rebased and fixed up the commit graph. I couldn't find anything else that would be affected from outside the class. The bug occurred in a beta version of my mod, so the fix isn't overly urgent, but sooner is of course better than later.

bonii-xx commented:
Great, I'll throw out a small release then. In case I forget, ping me some time down the road.
